core_impl/src/test/java/io/opencensus/implcore/stats/ViewManagerImplTest.java

```diff
@@ -581,17 +699,54 @@ public void testMultipleViewsDifferentMeasures() {
   }
 
   @Test
+  public void testMultipleViewsDifferentMeasures2() {
```

Comment:
As above, I think this method should share a parameterized helper method with testMultipleViewsDifferentMeasures.  I also think the name should describe how it differs.

Reply:
Done.